src/Illuminate/Log/LogManager.php

```diff
@@ -747,6 +759,19 @@ public function log($level, $message, array $context = []): void
         $this->driver()->log($level, $message, $context);
     }
 
+    /**
+     * Set the application instance used by the manager.
+     *
+     * @param  \Illuminate\Contracts\Foundation\Application  $app
+     * @return $this
+     */
+    public function setApplication($app)
+    {
+        $this->app = $app;
+
+        return $this;
+    }
+
```

Comment:
We need to refresh the instance on the log manager for Context to work with Octane: https://github.com/laravel/octane/pull/826